daemon/api_systems.go

```diff
@@ -105,10 +105,18 @@ func getAllSystems(c *Command, r *http.Request, user *auth.UserState) Response {
 }
 
 type oneSystemResponse struct {
+	// First part is designed to look like `client.System` - the
+	// only difference is how the model is represented
+	Current bool                       `json:"current,omitempty"`
+	Label   string                     `json:"label,omitempty"`
 	Model   clientutil.ModelAssertJSON `json:"model,omitempty"`
```

Comment:
as I mentioned I think, this could be directly  the headers, not using clientutil.ModelAssertJSON. ModelAssertJSON is like that because is used also for serial as well which might have a body